### PR TITLE
HW1 (ea2905): adding proposal for custoplayer, adding jotai project

### PR DESCRIPTION
**Pull request by Etesam913**

### CODE REVIEW

File: custoplayer.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# custoplayer
```
Comment (timkpaine) [inline custoplayer.md:1]: Move to `projects/javascript`

File: jotai.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# jotai
```
Comment (timkpaine) [inline jotai.md:1]: Move to `projects/javascript`

**Review by timkpaine — DISMISSED** [review]

File: jotai.md (hunk 2 above)
Comment (Etesam913) [reply jotai.md]: Done 👍🏽

File: custoplayer.md (hunk 1 above)
Comment (Etesam913) [reply custoplayer.md]: Done 👍🏽

**Comment by timkpaine** [comment]
lgtm

**Comment by Etesam913** [comment]
Is there something else that you want me to do?

**Pull request closed without merging**